drivers/clock_control/clock_agilex_ll.c

```diff
@@ -0,0 +1,392 @@
+/*
+ * Copyright (c) 2019-2021, Intel Corporation. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <arch/cpu.h>
+#include <drivers/clock_control/clock_agilex_ll.h>
+#include <errno.h>
+#include <soc.h>
+#include <socfpga_handoff.h>
+#include <socfpga_system_manager.h>
+
+uint32_t wait_pll_lock(void)
+{
+	uint32_t data;
+	uint32_t count = 0;
+
+	do {
+		data = mmio_read_32(CLKMGR_OFFSET + CLKMGR_STAT);
+		count++;
+		if (count >= 1000)
+			return -ETIMEDOUT;
```

Comment:
you are returning `uint32_t`

Reply:
I will update the function to return 'int'. Same for the next function.

Reply:
A single line statement for if-block needs curly brackets. Please add.